src/ng2-smart-table/components/cell/cell-types/cell-textarea.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Output, EventEmitter, Input } from '@angular/core';
+
+import { Cell } from '../../../lib/data-set/cell';
+import { DefaultCellType } from './default-cell-type';
+
+@Component({
+  selector: 'cell-type-textarea',
+  template: `
+    <textarea [ngClass]="inputClass"
+              class="form-control"
+              [(ngModel)]="cell.newValue"
+              [name]="cell.getColumn().id"
+              [disabled]="!cell.getColumn().isEditable"
+              (click)="onClick($event)"
```

Comment:
Hi, thanks a lot for your review and excuse me for all the other code-formatting mistakes.

Regarding this comment, I think all those bindings were already there when I picked up the project, would you prefer if I create some methods in the components and bind to them like

_Template_
```
[name]="getId()"
```
_Component_
```
getId() {
  return this.cell.getColumn().id;
}
```
?

Regarding the "build with AOT" do you refer to running `npm run build:aot` before submitting the PR? If not, could you point me in the right direction? Thanks a lor